client/layout/masterbar/logged-out.jsx

```diff
@@ -149,17 +150,23 @@ class MasterbarLoggedOut extends Component {
 		}
 
 		return (
-			<Masterbar>
-				<Item className="masterbar__item-logo">
-					<WordPressLogo className="masterbar__wpcom-logo" />
-					<WordPressWordmark className="masterbar__wpcom-wordmark" />
-				</Item>
-				<Item className="masterbar__item-title">{ title }</Item>
-				<div className="masterbar__login-links">
-					{ this.renderSignupItem() }
-					{ this.renderLoginItem() }
-				</div>
-			</Masterbar>
+			<>
+				{ currentRoute.startsWith( '/plugins' ) ? (
+					<PluginNavbarHeader />
```

Comment:
Seems that `PluginNavbarHeader` is a component that has nothing in common with `MasterbarLoggedOut` where it's rendered. Doesn't use any of its props or helper methods. A better place to render it would be [this condition](https://github.com/Automattic/wp-calypso/blob/trunk/client/layout/logged-out.jsx#L75-L110) in `LayoutLoggedOut`. Add one more condition, checking for the `/plugins` route, to choose between the many possible masterbars.